data-center-and-power-discussion.md

```diff
@@ -4,42 +4,53 @@
 
 ### Open Telemetry
 
-* https://opentelemetry.io/
-* https://docs.gprofiler.io/
-* https://github.com/hubblo-org/scaphandre
 
 ## Node-Level Variables
 
 ### Power
-* Kubernetes Power Manager:  https://github.com/intel/kubernetes-power-manager
+* Kubernetes Power Manager[^k8s-power-manager]
 
 ###
 
 ## Scheduling Options in Kubernetes
 
 ### On-demand 
 
-* Power Driven Scheduling and Scaling with CPU telemetry in K8s: https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling/docs/power
-
+* Power Driven Scheduling and Scaling with CPU telemetry in K8s[^platform-aware-scheduling] 
 ### Batch scheduling according to power costs (carbon, money, et cetera)
 
 # Current Research/Initiaives
 
 ## Current Sustainability Initiatives
-* Equinix: https://www.equinix.com/newsroom/press-releases/2022/04/equinix-prices-1-2-billion-of-green-bonds-in-its-fourth-offering-to-advance-sustainability-initiatives
-* Etsy - Cloud carbon footprint.org: https://www.cloudcarbonfootprint.org/docs/methodology/
-* LF Energy:  https://www.lfenergy.org/
-* Energy Efficient High Performance Computing Working Group - https://eehpcwg.llnl.gov/
+* Equinix[^equinix]
+* Etsy - Cloud carbon footprint.org[^ccf]
+* LF Energy[^lfenergy]  
+* Energy Efficient High Performance Computing Working Group[^llnl]
 
 ## Current Papers:
 
-* Emissions - Global Energy and CO2 Status Report 2019:  https://www.iea.org/reports/global-energy-co2-status-report-2019/emissions
-* EU Greenhouse Emission Intensity:  https://www.eea.europa.eu/ims/greenhouse-gas-emission-intensity-of-1
+* Emissions - Global Energy and CO2 Status Report 2019[^iea-emission] 
+* EU Greenhouse Emission Intensity[^europa-ghg]
 
 ### HPC Specific models to learn from:
 
-* Metrics for Evaluating Energy Saving Techniques for Resilient HPC Systems: https://www.osti.gov/servlets/purl/1140455
+* Metrics for Evaluating Energy Saving Techniques for Resilient HPC Systems [^osti]: 
 
 ## Repos:
 
-* Energy Efficiency of Languages:  https://github.com/greensoftwarelab/Energy-Languages
+* Energy Efficiency of Languages[^energy-lanaguage]
+
+# References
```

Comment:
I'd like us to group "current projects" and "reference papers."  I ran out of time yesterday, but having a brief summary before each paper would be useful too.  Thoughts?